src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +56,31 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        AuthenticationService authenticationService = new AuthenticationServiceImpl(userService);
+
+        User newRegisteredUser = null;
+        try {
+            System.out.println("=========================================");
+            newRegisteredUser = authenticationService.register("mary@mail.com", "111111");
+        } catch (RegistrationException e) {
+            throw new RegistrationException("Can't register user.",e);
```

Comment:
Throwing a `RegistrationException` here is redundant because it is already declared in the method signature. Consider logging the exception instead of re-throwing it.

Reply:
ok, done